packages/views/screens/router/routes.js

```diff
@@ -71,9 +71,10 @@ export default {
     forbiddenTokens: [],
   },
   manageAccounts: {
-    path: '/account',
+    path: '/',
```

Comment:
@soroushm the default route is not finalised yet, for now i understand we do not have sign in anymore and this is ok. But we might want to change this at later point in time based on new design changes to dashboard